.github/workflows/ci.yaml

```diff
@@ -0,0 +1,21 @@
+name: CI
+on: push
```

Comment:
Let's trigger the workflow on `push`, `pull_request`, and `workflow_dispatch` events for `master` as well as `develop` branches.

Reply:
For pull_request i'll filter to master and develop. As for push, I'll leave without branch filter so that tests will run on every commit which helps to test in our own forks before creating PR.